game/scripts/vscripts/items/cave_rune_hasted_doubledamage.lua

```diff
@@ -0,0 +1,26 @@
+LinkLuaModifier("modifier_rune_haste", LUA_MODIDIER_MOTION_NONE)
+LinkLuaModifier("modifier_rune_doubledamage", LUA_MODIDIER_MOTION_NONE)
+
+item_cave_rune_hasted_doubledamage = class(ItemBaseClass)
+
+function item_cave_rune_hasted_doubledamage:GetIntrinsicModifierName()

```

Comment:
does it work? I think we can't set multiple modifiers with this.
Since you add them on spellStart you can remove this.

Reply:
According to LUA, it should work, but I am not sure about DOTA. This needs a lot of review atm

Reply:
I does not work, Dota expects only one string ad result with the modifier, If it worked we would not have to call AddNewModifier on spell start because dota would do that for us.

Reply:
IntrinsicModifier is the wrong thing for this in any case. For items, the intrinsic modifier is a modifier that is added whenever the unit has the item active in inventory and removed whenever the item is backpacked or otherwise leaves the inventory.